exchange/exchangetest/mediatypepricegranularity-native-and-video.json

```diff
@@ -0,0 +1,324 @@
+{
+    "incomingRequest": {
+        "ortbRequest": {
+            "id": "some-request-id",
+            "site": {
+                "page": "test.somepage.com"
+            },
+            "imp": [
+                {
+                    "id": "my-imp-id",
+                    "video": {
+                        "mimes": [
+                            "video/mp4"
+                        ]
+                    },
+                    "ext": {
+                        "prebid": {
+                            "bidder": {
+                                "appnexus": {
+                                    "placementId": 1
+                                }
+                            }
+                        }
+                    }
+                },
+                {
+                    "id": "imp-id-2",
+                    "video": {
+                        "mimes": [
+                            "video/mp4"
+                        ]
+                    },
+                    "ext": {
+                        "prebid": {
+                            "bidder": {
+                                "appnexus": {
+                                    "placementId": 1
+                                }
+                            }
+                        }
+                    }
+                }
+            ],
+            "test": 1,
+            "ext": {
+                "prebid": {
+                    "targeting": {
+                        "includebrandcategory": {
+                            "primaryadserver": 1,
+                            "publisher": "",
+                            "withcategory": true
+                        },
+                        "pricegranularity": {
+                            "precision": 2,
+                            "ranges": [
+                                {
+                                    "min": 0,
+                                    "max": 20,
+                                    "increment": 2
+                                }
+                            ]
+                        },
+                        "mediatypepricegranularity": {
+                            "banner": {
+                                "precision": 3,
+                                "ranges": [
+                                    {
+                                        "max": 20,
+                                        "increment": 4.5
+                                    }
+                                ]
+                            },
```

Comment:
For my understanding, because i've seen this in other `mediattypepricegranualrity` tests, is in the filename it specifies `native` `and` video only, but in the file I see a `banner` price granularity specified as well. Why is this the case? Could the `banner` part of this be removed?

Reply:
Sure, idea here is to have configs for all media types. Native and video are bids that are returned. 
Test also tests that `req.ext.prebidmediattypepricegranualrity.mediatype.video` will be applied for video bid and `req.ext.prebidmediattypepricegranualrity.mediatype.native` will be applied to native bid. 
Regardless of what we have in `req.ext.prebidmediattypepricegranualrity.mediatype` correct config will be selected. 
I'm open to remove it and leave native and video configs only.